Simulator/Vertices/Neuro/AllIZHNeurons_d.cpp

```diff
@@ -171,7 +171,7 @@ void AllIZHNeurons::advanceVertices(AllEdges &synapses, void *allVerticesDevice,
    int blocksPerGrid = (vertex_count + threadsPerBlock - 1) / threadsPerBlock;
 
    // Advance neurons ------------->
-   advanceIZHNeuronsDevice<<<blocksPerGrid, threadsPerBlock>>>(
+   advanceIZHNeuronsDevice<<<blocksPerGrid, threadsPerBlock, 0, stream>>>(
```

Comment:
`stream` is a data member of the class; should be named `stream_`. Is this supposed to be distinct from the `AllEdges` stream?